pkgs/by-name/bi/bitwarden-desktop/fix-lock-files.diff

```diff
@@ -0,0 +1,282 @@
+diff --git a/apps/cli/package.json b/apps/cli/package.json
+index 2ec4e6f697..db5981b5ec 100644
+--- a/apps/cli/package.json
++++ b/apps/cli/package.json
+@@ -85,7 +85,7 @@
+     "multer": "1.4.5-lts.2",
+     "node-fetch": "2.6.12",
+     "node-forge": "1.3.1",
+-    "open": "10.1.2",
++    "open": "8.4.2",
+     "papaparse": "5.5.3",
+     "proper-lockfile": "4.1.2",
+     "rxjs": "7.8.1",
+diff --git a/apps/desktop/desktop_native/Cargo.lock b/apps/desktop/desktop_native/Cargo.lock
+index 05663ea7e0..eadd75e598 100644
+--- a/apps/desktop/desktop_native/Cargo.lock
++++ b/apps/desktop/desktop_native/Cargo.lock
+@@ -162,7 +162,7 @@ dependencies = [
+  "serde_repr",
+  "tokio",
+  "url",
+- "zbus 5.6.0",
```

Comment:
In https://github.com/bitwarden/clients/commit/fc9ce266ba60e489ad78eb919f7b439192af4ff1 basically the same change happened